pwiz_tools/Skyline/TestPerf/EncyclopeDiaSearchTutorialTest.cs

```diff
@@ -71,12 +70,12 @@ public void TestEncyclopeDiaSearchTutorial()
                     //"23aug2017_hela_serum_timecourse_wide_1f.mzML",
                 },
 
-                FinalTargetCounts = new[] { 317, 552, 552, 3922 },
+                FinalTargetCounts = new[] { 370, 725, 725, 5104 },
```

Comment:
Do we understand why these numbers get so much bigger?

Reply:
Not entirely. I had been thinking it was due to slight differences in the Prosit peaks and retention times predict with Koina, but a deeper comparison of the old prosit blib and the new koina one reveals some pretty extreme RT differences:
![image](https://github.com/ProteoWizard/pwiz/assets/5011937/6117cd98-a208-4183-adb2-c6092146c3d4)
The peak count differences are not nearly as extreme. Do you think the RT differences would explain EncyclopeDIA results? I need to dig into that to see what's going on.

Reply:
It would probably be good, then, to look at the RT regression plots in residuals mode to get a sense of how far from the predictions the detected peaks end up. It would definitely make a big difference if Koina improved its iRT prediction a lot. I found that Prosit's iRT predictions were generally twice as far away from the detected peaks as empirical values.

https://skyline.ms/_webdav/home/software/Skyline/%40files/2020-ASMS-Skyline-Prosit.pdf#page=11

It might be a good idea for us/me to rerun these datasets with Koina predictions. You can see on page 11 of the presentation how detections went way down with Prosit unless the chromatogram extraction window was doubled from 10 minutes to 20 minutes.

Reply:
Turns out the old PredictBatches function I implemented for EncyclopeDIA was never getting the iRTs right (the same peptide was getting different iRTs!). So we can't really compare the new results with the old. All I can do is say that the peptides are getting the correct iRTs now (into the blib) and it matches what Koina predicts when I use their python example code. I think maybe that EncyclopeDIA tutorial needs a critical and knowledgeable eye to go over it to see if any of the settings are unreasonable. You mentioned used a really big RT window for matching when using Prosit and it seems to be using quite a small window (based on the MacCoss lab EncyclopeDIA guide I followed, IIRC).